patches/v8/0010-Set-torque-generator-path-to-external-v8.-This-allow.patch

```diff
@@ -1,4 +1,4 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
This patch is no longer needed – the approach in https://chromium-review.googlesource.com/c/v8/v8/+/6111432 works with and without bzlmod, when building V8 directly and as a dependency.